src/planscape/goals/migrations/0001_initial.py

```diff
@@ -0,0 +1,240 @@
+                ("depth", models.PositiveIntegerField()),
+                ("numchild", models.PositiveIntegerField(default=0)),
+                ("name", models.CharField(max_length=128)),
+                ("path", models.CharField(max_length=512)),
+                (
+                    "created_by",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.RESTRICT,
+                        related_name="created_treatment_goal_categories",
+                        to=settings.AUTH_USER_MODEL,
+                    ),
+                ),
+                (
+                    "project",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.RESTRICT,
+                        related_name="treatment_goal_categories",
+                        to="projects.project",
+                    ),
+                ),
+            ],
+            options={
+                "verbose_name": "Treatment Goal Category",
+                "verbose_name_plural": "Treatment Goal Categories",
+            },
+        ),
+        migrations.CreateModel(
+            name="TreatmentGoal",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("created_at", models.DateTimeField(auto_now_add=True, null=True)),
+                ("updated_at", models.DateTimeField(auto_now=True)),
+                (
+                    "deleted_at",
+                    models.DateTimeField(
+                        help_text="Define if the entity has been deleted or not and when",
+                        null=True,
+                        verbose_name="Deleted at",
+                    ),
+                ),
+                ("uuid", models.UUIDField(db_index=True, default=uuid.uuid4)),
+                (
+                    "name",
+                    models.CharField(
+                        help_text="Name of the treatment goal. Equivalent to short_question_text.",
+                        max_length=128,
+                    ),
+                ),
+                (
+                    "long_question_text",
+                    models.CharField(
+                        help_text="this is a legacy field required to keep compatibility. most likely will be deprecated soon.",
+                        max_length=512,
+                        null=True,
+                    ),
+                ),
+                (
+                    "description",
+                    models.TextField(
+                        help_text="This field should contain the help text displayed as the box on the right, after you select the treatment goal.",
```

Comment:
We shouldn't directly reference the UI here (it can easily change and this will get out of date).
Maybe something like
"Describes to the user how the goal is set up, how it identifies the areas and the set of thresholds, as well as notes on usage" (or something along those lines :D)